src/invidious/yt_backend/extractors.cr

```diff
@@ -169,7 +169,12 @@ private module Parsers
       # When public subscriber count is disabled, the subscriberCountText isn't sent by InnerTube.
       # Always simpleText
       # TODO change default value to nil
+
       subscriber_count = item_contents.dig?("subscriberCountText", "simpleText")
+      if !subscriber_count || !subscriber_count.as_s.includes? " subscriber"
```

Comment:
```suggestion

      # Since youtube added channel handles, `VideoCountText` holds the number of
      # subscribers and `subscriberCountText` holds the handle, except when the
      # channel doesn't have a handle (e.g: some topic music channels).
      # See https://github.com/iv-org/invidious/issues/3394#issuecomment-1321261688
      if !subscriber_count || !subscriber_count.as_s.includes? " subscriber"
```